### ISSUE TITLE
Build fails on 10.6.8: error: 'SCARD_E_NO_READERS_AVAILABLE' undeclared

### ISSUE THREAD

**Issue by barracuda156** [OpenSC/OpenSC#2596]
### Problem Description

Identical failure when building with default `gcc-4.2` and the latest `gcc12`:
```
reader-pcsc.c: In function 'pcsc_to_opensc_error':
reader-pcsc.c:185:14: error: 'SCARD_E_NO_READERS_AVAILABLE' undeclared (first use in this function); did you mean 'SCARD_E_READER_UNAVAILABLE'?
  185 |         case SCARD_E_NO_READERS_AVAILABLE:
      |              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |              SCARD_E_READER_UNAVAILABLE
reader-pcsc.c:185:14: note: each undeclared identifier is reported only once for each function it appears in
make[3]: *** [libopensc_la-reader-pcsc.lo] Error 1
```

See: https://trac.macports.org/ticket/65709

**Comment by frankmorgner** [comment]
This has been fixed some time ago https://github.com/OpenSC/OpenSC/commit/548a2036328a288d24a48797498c3f76d6e351ef

**Comment by barracuda156** [comment]
> This has been fixed some time ago [548a203](https://github.com/OpenSC/OpenSC/commit/548a2036328a288d24a48797498c3f76d6e351ef)

Thank you, that works.
